### PR TITLE
update interface vs type aliases docs, to include features supported …

### PR DESCRIPTION
**Pull request by cryptoB0T**
…since ts 2.7

Fixes #
The docs should indicate that type aliases can be extended, as this is supported since 2.7.

### CODE REVIEW

**Comment by orta** [comment]
I tried looking through the release notes but I couldn't find anything about this feature, do you mean something [like this](https://www.typescriptlang.org/play/#code/C4TwDgpgBAsiAq5oF4oG8BQUoEMBcUARgPbEA2EOAdhgL4YaiSwgCiAHsBFQCYQ+JmqOIOgAydEQIBnYACcAllQDmUehj4BjMjjnRNxKrKgRO3PjwJwOXXv1ENTtiwDocGJ+f4vCGIA)?

```ts
type MyType = {
  a: boolean
}

type MyExtendedType = MyType & { b: string }

declare const extended: MyExtendedType

extended.a
extended.b
```

**Comment by cryptoB0T** [comment]
yes, this is correct. the docs appear to indicate that type aliases cannot be extended. it doesn't appear in the release notes but the TS compiler seems to support this functionality. is there something I'm missing?

File: pages/Advanced Types.md [hunk 1]
```diff
@@ -483,7 +483,8 @@ declare function aliased(arg: Alias): Alias;
 declare function interfaced(arg: Interface): Interface;
 ```
 
-A second more important difference is that type aliases cannot be extended or implemented from (nor can they extend/implement other types).
+Type aliases couldn't be extended or implemented from (nor could they extend/implement other types), in older versions of TypeScript. As of version 2.7, type aliases can be extended. Teams developing React applications sometimes choose to use `Type` aliases over `Interface`.
```
Comment (orta) [inline pages/Advanced Types.md]:
```suggestion
in older versions of TypeScript, type aliases couldn't be extended or implemented from (nor could they extend/implement other types). As of version 2.7, type aliases can be extended by creating a new intersection type e.g. `type Cat = Animal & { purrs: true }`.
```

**Comment by orta** [comment]
Yeah, I also was a little stumped - but that's fine, I've tried to update the PR to include that info with an example to make it more concrete 👍

**Comment by orta** [comment]
Thanks!

**Pull request merged**